### ISSUE TITLE
`no-eol-whitespace` does not see the whitespace at the end of a line a form feed ends

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#247]
Found by the review of [#244](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/244), which swept the plugin for readings that count line feeds and carriage returns alone. This one is not reached by that change and is as old as the rule.

### What minimal example or steps are needed to reproduce the bug?

Whitespace at the end of a line the file ends with a form feed:

```css
a { color: pink; \f}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/no-eol-whitespace": true
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json --fix
```

### What did you expect to happen?

`Unexpected whitespace at end of line` at `1:17`, and the space trimmed by `--fix`, which is what all three of the other spellings get: `a { color: pink; \n}`, `a { color: pink; \r}` and `a { color: pink; \r\n}` are each reported at that position and each come back trimmed. A form feed ends a line to every syntax this plugin reads through, so the same answer.

### What actually happened?

Nothing is reported and nothing is trimmed. The space stands at the end of a line, which is the one thing this rule is about.

### What is the cause?

The scan that finds where the lines end reads `EVERY_CR_OR_LF`, `/[\n\r]/gu`, in [`no-eol-whitespace`](https://github.com/stylelint-stylistic/stylelint-stylistic/blob/main/lib/rules/no-eol-whitespace/index.js), and a form feed matches nothing there, so the character is not an end of line to this rule and the whitespace in front of it is not at the end of one.

[#173](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/173), [#196](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/196), [#204](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/204), [#209](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/209) and [#244](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/244) each moved one such reading to the three characters that do end a line; `EVERY_LINE_BREAK`, `/[\n\r\f]/gu`, is already in `lib/regexps.js` under that name.

Two other readings of the same pair are different questions and stay where they are: `linebreaks` asks which of the two characters a file ends its lines with, a form feed being neither, and `OPENS_WITH_CR_OR_LF` in `no-empty-first-line` asks about the blank first line an editor writes.

**Comment by firefoxic** [comment]
One sentence of the body above is wrong, and the measurement that shows it is in the census of [#250](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/250).

It says `OPENS_WITH_CR_OR_LF` in `no-empty-first-line` is a different question and stays where it is. It is not: that rule reports a file opening on a line feed and a file opening on a bare carriage return alike, and passes over one opening on a form feed.

```
"\na { color: pink; }"   -->  Unexpected empty line at the beginning of source, trimmed by --fix
"\ra { color: pink; }"   -->  the same
"\fa { color: pink; }"   -->  nothing at all
```

So the name is narrow the way `EVERY_CR_OR_LF` is, and the exclusion its comment carried was not an answer but half of one. Nothing about this issue changes — `no-eol-whitespace` is still what it is about — only the closing paragraph's claim that the other two readings of the pair are settled. `linebreaks` still is; `no-empty-first-line` is not.
